_ext/eclipse-base/src/main/java/com/diffplug/spotless/extra/eclipse/base/osgi/EclipseBundleLookup.java

```diff
@@ -16,12 +16,9 @@
 package com.diffplug.spotless.extra.eclipse.base.osgi;
 
 import org.osgi.framework.Bundle;
-import org.osgi.service.packageadmin.ExportedPackage;
-import org.osgi.service.packageadmin.PackageAdmin;
-import org.osgi.service.packageadmin.RequiredBundle;
```

Comment:
What's up with this change?

Reply:
Sorry, been lazy. Changed this because of deprecation warnings. But I looked it up again. Was actually a bug in an Open-JDK version I used on one system. The standard states that the `@SuppressWarnings("deprecation")` on the outer class should be sufficient. Will undo it.